### ISSUE TITLE
[BB] Shared EXP doesn't work with NPCs

### ISSUE THREAD

**Issue by toiletman74** [fuzziqersoftware/newserv#565]
**Describe the bug**
If an NPC kills an enemy without the player dealing any damage to the enemy, EXP is not rewarded.

**To reproduce**
Fill in steps to reproduce the behavior, such as:
1. Start any quest with an NPC in it, Warrior's pride, Gallon's plan, etc...
2. Have the NPC kill an enemy without the player dealing any damage to that enemy
3. No exp is rewarded

**Game version(s) (choose one or more of the following):**
BB

**Comment by nolrinale** [comment]
This is correct behavior, for the game the NPC acts like just another player, if you dont at least tag the monster once and the NPC kills it, you wont get the EXP

So at the very least hit the monster once and let the NPC kill it afterwards

**Comment by fuzziqersoftware** [comment]
Correct, this is not a bug. If an NPC kills a monster that you did tag and you don't get EXP, that would be a bug - if that happens, please reopen this.

**Comment by fuzziqersoftware** [comment]
Oh wait - I misunderstood this. This is indeed a bug. Shared EXP refers to automatically-shared EXP, which is an option on the server. With the option on, you should get EXP even for enemies you didn't tag, if you're in the same area when they're killed. I'll look into this.

**Comment by toiletman74** [comment]
Yeah I meant the it doesn't work with the shared EXP option, I should've specified
